aries-site/src/pages/templates/page-layouts.js

```diff
@@ -16,10 +50,128 @@ const PageLayouts = () => {
         description={pageDetails.seoDescription}
         canonicalUrl="https://design-system.hpe.design/templates/page-layouts"
       />
-      <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
-      </ContentSection>
-      <ComingSoon />
+      <ResponsiveContext.Consumer>
+        {size => (
```

Comment:
Hooks implementation instead? Either way works, just a code consistency question.

Reply:
Let's do it : )

Reply:
Done.